### ISSUE TITLE
GetSpriteInfo 参数写反了？

### ISSUE THREAD

**Issue by duanxu520** [coding2233/TextInlineSprite#56]
![image](https://user-images.githubusercontent.com/4090262/124588907-e2dbcf80-de48-11eb-985c-3098188490b4.png)
这里是不是写反了？

**Comment by wolf-aowu** [comment]
确实。我使用 Unity 自带的 Sprite Editor 也是按行和列切的方法切图，并且把 行数 设置成 正常行数的 2 倍，也就说一小块图片应该被分成上下两半，Unity 自带的是正确的，而插件就不正确了，安你说的改了以下就正确了。
但还有一个问题，如下图所示：
插件切出的图
![插件切图问题](https://user-images.githubusercontent.com/44164171/144739802-d61f56c7-58c5-4df1-b5a3-8edc71afc121.png)
官方切出的图
![插件切图问题（官方）](https://user-images.githubusercontent.com/44164171/144739823-5b27f1ef-abe4-40d7-99b4-f82623ea9651.png)
我正在查找是什么原因
